cl_manycore/libraries/bsg_manycore_driver.h

```diff
@@ -144,9 +144,13 @@ int hb_mc_all_host_req_complete(uint8_t fd);
 int hb_mc_get_config(uint8_t fd, hb_mc_config_id_t id, uint32_t *cfg);
 uint8_t hb_mc_get_manycore_dimension_x();
 uint8_t hb_mc_get_manycore_dimension_y();
+uint8_t hb_mc_get_host_intf_coord_x();
+uint8_t hb_mc_get_host_intf_coord_y();
 
 int hb_mc_fifo_finish(uint8_t fd);
+
```

Comment:
Was it necessary to add this whitespace?

Reply:
ha! happened by mistake :D I added a deprecated tag to each one, but it generated 20 warnings on compile so I removed them but forgot to remove the whitespace. This brings me to a philosophical question, is it better to remove the deprecated functions all together or leave them at the cost of generating 100 warnings? They only reason they were left there in the first place was to make sure the cuda API doesn't break all at once, but now that we don't need them anymore should I remove them?

Reply:
Deprecated tags. Then we also know what is deprecated